source/chapters/user_interface.rst

```diff
@@ -620,14 +620,24 @@ Headphone and Master Mix Controls
   This knob allows you to adjust the :term:`balance` (left/right orientation) of
   the master output.
 
-**Volume Knob**
-  The Volume Knob controls the overall volume of the master output. Adjust
+**Gain Knob**
+  The Gain Knob controls the overall gain of the master output. Adjust
   this knob so that the :ref:`Master VU meters <interface-vu-meter>` are just
   at the peak.
```

Comment:
are just below the yellow region. 
This should offer enough headroom for mixing and applying effects. 
This matches approximately a recommended target Loudness of -16 LUFS for broadcasting.

(The yellow area it the boom region, used in movies for explosions ans such)  

Related:
https://bugs.launchpad.net/mixxx/+bug/1429760
We may also consider replace the legacy VU meters with modern loudness meters with LUFS scale.